JT/15_how_to_identify_xpaths/How_To_Identify_XPaths_README.adoc

```diff
@@ -0,0 +1,741 @@
+= How to identify XPaths. Tools for XPaths. Cheat sheet
```

Comment:
please update link for this theme in Study Materials Intro.adoc file

Reply:
✔️Updated